documentation/onboard-dpg-in-sdkautomation/.net/README.md

```diff
@@ -17,5 +17,6 @@ require:
  - specification/<RPName>/dataplane/readme.md
 ```
 ~~~
+- `title`: If it's already defined in spec readme.md, you don't need to define it here again. (By default, there is no `tag` defined in spec readme.md in multi client scenario.)
```

Comment:
Isn't the title already in the spec as well?

Reply:
No. In multi client scenario, some languages may have different title in each sub client. If the `title` is defined in readme.md, then we cannot specify the subclient's title. So it's not defined in readme.md by default in multi client scenario.